proposal/JieJhih/README.md

```diff
@@ -0,0 +1,58 @@
+Packet Analyzer
+==========================
+Packet Analyzer is a tool to capture the packets and make it visualize.
+To capture the packets I use C++ to write a eBPF program, which makes the speed of capture packets is really fast. (Please follow [Packet flow in Netfilter and General Networking](https://upload.wikimedia.org/wikipedia/commons/3/37/Netfilter-packet-flow.svg)) 
+After get a bunch of packets, we use python to visualize those packets.
+
+Basic Information
+=================
+
+GitHub repository:
+https://github.com/JieJhih/Packet-Analyzer.git
+
+Problem To Solve
+================
+
+A server will receive a bunch of packets every day, sometimes system administrator need to analyze each kind of packet (TCP? UDP? Specific IP?) occupy how many percent in daily. Using eBPF program is faster than other userspace tools, cooperate with Python could make data visible.
+
+Perspective users
+=================
+1. System administrator who want to maintain server's health.
+2. Data scientists who want to analyze the client agent, and packets peak or off-peak hours.
+
+System architecture
+=================
+
+1. Capture the packets from eBPF program.
+2. Send packets information to Python.
+3. Organize a bunch of information of packets.
+4. Show the statistic using Python GUI tool.
+
+API decription
+=================
+1. Packet pool buffers.
```

Comment:
I think it makes sense to pool the (packet) information.  But this seems to contradict to your second bullet in System architecture.